README.md

```diff
@@ -16,33 +68,37 @@
 
 Install both packages using [official installer](https://nodejs.org/en/download/). After installation make sure both packages have proper version by running `node -v` and `npm -v`. To upgrade invalid npm version run `npm install npm@latest -g`
 
-## Install
+## Using from the command-line interface (CLI)
+
+### Install the CLI
+
+To use it as CLI, install generator globally:
 
 ```bash
 npm install -g @asyncapi/generator
 ```
 
-Or just use Docker:
+### Update the CLI
 
-```bash
-docker run --rm -it \
--v [ASYNCAPI FILE LOCATION]:/app/asyncapi.yml \
--v [GENERATED FILES LOCATION]:/app/output \
-asyncapi/generator [COMMAND HERE]
-# Example that you can run inside generator directory after cloning this repository. First you specify mount in location of your AsyncAPI file and then you mount in directory where generation result should be saved.
-docker run --rm -it \
--v ${PWD}/test/docs/streetlights.yml:/app/asyncapi.yml \
--v ${PWD}/output:/app/output \
-asyncapi/generator -o ./output asyncapi.yml @asyncapi/html-template --force-write
-```
+You might want to update your local installation of generator for different reasons:
 
-## Usage
+* You want latest generator to have it's latest features. Perform usual installation and in case you had generator installed already, it will upgrade to latest available:
+    ```bash
+    npm install -g @asyncapi/generator
+    ```
+* You want specific version of the generator because your template might not be compatible with latest generator. Check [what version you need](https://github.com/asyncapi/generator/releases) and perform installation, specifying exact version with `@` character:
```

Comment:
```suggestion
* You want a specific version of the generator because your template might not be compatible with the latest generator. Check [what version you need](https://github.com/asyncapi/generator/releases) and perform installation, specifying the exact version with the `@` character:
```